.gitignore

```diff
@@ -1,6 +1,6 @@
 npm-debug.log
 node_modules/
-dist/
+dist/browser-extension/
```

Comment:
Why not exclude all contents in the dist directory?

Reply:
We need to submit `dist/userscript/index.js` to github in order to generate a link from [jsdelivr](https://www.jsdelivr.com/github)

Reply:
I see, maybe we should add `dist/electron` and `dist/tauri`, or put the contents of `userscript` into another directory.

Reply:
forgot that